website/docs/admin_settings_local.md

```diff
@@ -16,13 +16,21 @@ OpenPype stores some of it's settings and configuration in local file system. Th
 ## Categories
 
 ### OpenPype Mongo URL
+The **Mongo URL** is the database URL given by your Studio. More details [here](artist_getting_started.md#mongodb).
 
 ### General
+**OpenPype Username** : enter your username (it can also take by default the computer session username). It signs your actions on **OpenPype**.
```

Comment:
Admittedly this is unclear to me. How does it sign my actions on OpenPype? What does that influence?

Here is it a bit tweaked. Note that I'm not sure if my example is actually correct and whether it does that.

```suggestion
**OpenPype Username** : enter your username (if not provided, it uses computer session username by default). This username is used to sign your actions on **OpenPype**, for example the "author" on a publish.
```